client/blocks/reader-full-post/_content.scss

```diff
@@ -197,6 +197,29 @@
 			}
 		}
 	}
+
+	sup, sub {
+		vertical-align: baseline;
+		position: relative;
+		font-size: 0.83em;
```

Comment:
This is the default font size for `sup` and `sub` in most browsers (see https://github.com/Automattic/wp-calypso/pull/2937#r51473415).